lib/browsers.js

```diff
@@ -28,10 +28,13 @@ var browserDefinitions = {
       'firefox-developer': ['firefox-developer']
     }
   },
-  phantomjs: {
+  phantom: {
```

Comment:
If you're considering that it's an incompatible change from `browser-launcher2`, then yes, absolutely it is a change. Though `james-browser-launcher` is _close_ to being a drop-in replacement, it isn't entirely, and we're considering API changes OK, so long as we document them in the `CHANGELOG`